meta-balena-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -44,12 +44,6 @@ do
 done
 echo "Docker started."
 
-# Pull in arch specific hello-world image and tag it balena-healthcheck-image
-echo "Pulling ${HELLO_REPOSITORY}:latest..."
-docker pull --platform "${HOSTAPP_PLATFORM}" "${HELLO_REPOSITORY}"
-docker tag "${HELLO_REPOSITORY}" balena-healthcheck-image
-docker rmi "${HELLO_REPOSITORY}"
-docker save balena-healthcheck-image > ${BUILD}/balena-healthcheck-image.tar
```

Comment:
There's more cleanup to do here, specifically:
* remove the `HELLO_REPOSITORY` variables definitions from [`meta-balena-common/recipes-containers/docker-disk/docker-disk.inc`](https://github.com/balena-os/meta-balena/blob/c34abd0168622b7766004de8a27f0e96f556b035/meta-balena-common/recipes-containers/docker-disk/docker-disk.inc#L46)
* Remove passing the `HELLO_REPOSITORY` enviromental variable in https://github.com/balena-os/meta-balena/blob/c34abd0168622b7766004de8a27f0e96f556b035/meta-balena-common/recipes-containers/docker-disk/docker-disk.bb#L63